howso/utilities/feature_attributes/pandas.py

```diff
@@ -468,7 +478,8 @@ def _infer_floating_point_attributes(self, feature_name: str) -> dict:
         if self.data[feature_name].isna().sum() < n_cases:
 
             # determine if nominal by checking if number of uniques <= 2
-            if self.data[feature_name].nunique() <= 2 and n_cases > 10:
+            if self.data[feature_name].nunique() <= 2 and n_cases > 10 and \
+                feature_type_override not in ('continuous', 'ordinal'):
```

Comment:
Seems like this added if logic is not needed? If its nominal it returns immediately at the top. As is this breaks existing functionality since it will not pass when feature_type_override is None.